### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by alekaizer**
Add the ACL option to specify the ACL policy of the files uploaded.
Defautl: 'private'

In my case I needed my pictures to be seen by everyone, but by default AMAZON ACL is private, so I had to go through all the pictures and change their permissions which you can understand was a nightmare.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -33,6 +33,9 @@ function S3Storage (opts) {
   if (!opts.accessKeyId) throw new Error('accessKeyId is required')
   if (!opts.region) throw new Error('region is required')
   if (!opts.dirname) throw new Error('dirname is required')
+  if (!opts.acl) console.log('--------------------------------------------------------------\n'+
+                              '[MULTER S3] ACL is not specified, will used private by default\n'+
+                              '--------------------------------------------------------------')
```
Comment (LinusU) [inline index.js]: I'm not a fan of this...

**Comment by LinusU** [comment]
I think we should either let `acl` silently default to `private` (as it is now I believe) or throw an error when it's not provided (like the other options).

Which one do you prefer @badunk?

Let's get this in for 2.0 as well.

File: index.js (hunk 1 above)
Comment (alekaizer) [inline index.js]: @LinusU you're not fan of the warning or the ACL adding ?

Comment (LinusU) [inline index.js]: The warning

Comment (alekaizer) [inline index.js]: @LinusU  Okay, removed
My thought was to warn that the default *acl * was private, but yeah that can be notified in the doc

**Comment by badunk** [comment]
I actually think we shouldn't specify the default ourselves or default to `"{}"` according to the s3 documentation - basically let s3 handle whatever default it wants to in their library:

http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html
> params (Object) (defaults to: {}) —
> ACL — (String) The canned ACL to apply to the object. Possible values include:
> "private"
> "public-read"
> "public-read-write"
> "authenticated-read"
> "aws-exec-read"
> "bucket-owner-read"
> "bucket-owner-full-control"

Since we moved to using the official s3 for javascript, perhaps we should just pass/merge the entire config object along and only validate what's necessary.  But we can do that in another PR :)

Do you see any problems with that @LinusU?

**Comment by alekaizer** [comment]
Oh, well I see what you mean, and sounds good, doing another PR then :)

**Comment by LinusU** [comment]
That sounds good

**Comment by badunk** [comment]
Check out `1.5.0`

**Pull request closed without merging**